application/core/filters.py

```diff
@@ -9,10 +9,18 @@
 import jsonpickle
 from bs4 import BeautifulSoup
 from slugify import slugify
-
 from uritemplate import URITemplate
 
 
+def to_sentence_case(string):
```

Comment:
@paulmsmith Jinja has this built in https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.capitalize

Reply:
Oh heck.. Will refactor that.